pkg/controller/storagecluster/storagecluster_controller.go

```diff
@@ -41,19 +41,21 @@ func Add(mgr manager.Manager) error {
 }
 
 func (r *ReconcileStorageCluster) initializeImageVars() error {
-	r.cephImage = os.Getenv("CEPH_IMAGE")
-	r.noobaaCoreImage = os.Getenv("NOOBAA_CORE_IMAGE")
-	r.noobaaDBImage = os.Getenv("NOOBAA_DB_IMAGE")
+	r.images = map[string]string{
+		"ceph":        os.Getenv("CEPH_IMAGE"),
+		"noobaa-core": os.Getenv("NOOBAA_CORE_IMAGE"),
+		"noobaa-db":   os.Getenv("NOOBAA_DB_IMAGE"),
```

Comment:
Can we define these keys as constants? Since it's referenced at multiple places that'll probably help us limit the changes if we ever refactor this.

Reply:
I believe that after the change, this is the only place where we refer to these env vars.
The only other places I could found are shell scripts were moving to constants is irrelevant

Reply:
What about the places where you are updating `r.Images["ceph"]` etc?

Reply:
Ok, I understand now. 
You meant `ceph`, `noobaa-core` and `noobaa-db`.
I was thinking you were referring to `CEPH_IMAGE`, `NOOBAA_CORE_IMAGE` and `NOOBAA_DB_IMAGE`

In that case, we can do constants for the keys in order to deflate the value range.

Reply:
done